scrapers/JavLand.yml

```diff
@@ -0,0 +1,54 @@
+name: "Jav.Land"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - jav.land
+    scraper: sceneScraper
+movieByURL:
+  - action: scrapeXPath
+    url:
+      - jav.land
+    scraper: movieScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: &infoSel //table[starts-with(@class,"videotextlist ")]
+    scene:
+      Title: &titleSel $info/tbody/tr[2]/td[2]/text()
```

Comment:
I am not a fan of using [1,2,3], you can use instead:
`//td[strong[text()="DVD ID:"]]/following-sibling::td/text()`
It work for other field too

Reply:
OK, I got it. Will fix it today.